HybridManagement/HybridManagement.psd1

```diff
@@ -84,13 +84,14 @@
         "Get-AzStorageKerberosTicketStatus",
         "Test-AzStorageAccountADObjectPasswordIsKerbKey",
         "Update-AzStorageAccountADObjectPassword",
-        "Join-AzStorageAccountForAuth", 
+        "Join-AzStorageAccount", 
         "Invoke-AzStorageAccountADObjectPasswordRotation",
         "Expand-AzResourceId",
         "Compress-AzResourceId",
         "Get-AzCurrentAzureADUser",
         "Test-AzPermission",
-        "Assert-AzPermission"
+        "Assert-AzPermission",

```

Comment:
Is this intended to be called by the customer to troubleshoot access or would you want to call it before attempting to list the keys in the "join" flow?  If the first one, is it already added into documentation?

Reply:
I intend for this cmdlet to be used by other cmdlets, but the logic is new, so I don't want to add it to our AD cmdlets until I'm confident in the implementation. I would also like to add an Assert-ADPermission cmdlet so we can ensure we have all the permissions we need before attempting to do actions in our cmdlets.